src/AutoRest.CSharp/Common/Input/InputTypes/Serialization/TypeSpecInputModelTypeConverter.cs

```diff
@@ -60,39 +65,70 @@ public static InputModelType CreateModelType(ref Utf8JsonReader reader, string?
 
                 if (reader.GetString() == nameof(InputModelType.Properties))
                 {
-                    model = CreateInputModelTypeInstance(id, name, ns, accessibility, deprecated, description, usageString, discriminatorValue, discriminatorPropertyName, baseModel, properties, resolver);
+                    model = CreateInputModelTypeInstance(id, name, ns, accessibility, deprecated, description, usageString, discriminatorValue, discriminatorPropertyName, baseModel, properties, isLowConfident, isNullable, resolver);
                     reader.Read();
                     CreateProperties(ref reader, properties, options);
-                    if (reader.TokenType != JsonTokenType.EndObject)
-                    {
-                        throw new JsonException($"{nameof(InputModelType)}.{nameof(InputModelType.Properties)} must be the last defined property.");
-                    }
+                    //if (reader.TokenType != JsonTokenType.EndObject)
```

Comment:
Any future use of them?  Can we remove those code directly?

Reply:
These are experimental changes. Will revert in the future